openapi/oh.yaml

```diff
@@ -8117,30 +8085,31 @@ components:
                         height:
                           type: number
                           format: double
-                        size:
+                        empty:
+                          type: boolean
+                        location:
                           type: object
                           properties:
-                            width:
+                            x:
                               type: number
                               format: double
-                            height:
+                            "y":
```

Comment:
Why is `y` in quotes and `x` isn't?   It seems consistent that way through the file.

Reply:
I noticed that but really I don't know... this yamel is generated by the `springdoc-openapi` maven plugin.
but if you launch API and open `http://localhost:8080/v3/api-docs.yaml` it is same way 🤷‍♂️